src/server/index.ts

```diff
@@ -244,7 +244,7 @@ export class DocumentDriveServer implements IDocumentDriveServer {
             if (isDocumentDrive(document)) {
                 await this.storage.addDriveOperations(
                     drive,
-                    operations as Operation<DocumentDriveAction | BaseAction>[], // TODO check?
+                    operations, // TODO check?
```

Comment:
this was done by the linter's auto fix because "the type cast does not change the type". we can re-add this if its needed for wip